hhs_oauth_server/settings/base.py

```diff
@@ -378,7 +379,7 @@
 # IF /testclient fails because the server is running without a certificate
 # eg. on your local machine. You need to un-comment the following line:
 
-os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = '1'
+# os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = '1'
```

Comment:
Can you please remove code entirely rather than commenting it out? There's no value in having the commented out version here, and it clogs up our codebase and makes it harder to figure out what's going on.